programmers/난이도별/level01.2016년/develop-sell.js

```diff
@@ -0,0 +1,19 @@
+function solution(a, b) {
+  var answer = '';
+  const day_week = ['SUN','MON','TUE','WED','THU','FRI','SAT'];
+  const month_day = [31,29,31,30,31,30,31,31,30,31,30,31];
+  let total_day = 0;
+  for(let i= 0; i < a-1; i++){  //for(let i= 1; i < a; i++) 로 작성했었음
+      total_day += month_day[i];
+  }
+  total_day += b;
+  answer = day_week[(total_day + 4) % 7];
+  
+  return answer;
```

Comment:
```suggestion
  return day_week([ ...Array(a-1).keys() ].reduce((sum, key) => sum + month_day[key], b + 4) % 7);
```
이렇게 표현할 수 있습니다.
[Array.Prototype.reduce](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce)에 대해 한 번 공부해보세요!